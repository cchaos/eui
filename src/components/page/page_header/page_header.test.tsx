/*
 * Licensed to Elasticsearch B.V. under one or more contributor
 * license agreements. See the NOTICE file distributed with
 * this work for additional information regarding copyright
 * ownership. Elasticsearch B.V. licenses this file to you under
 * the Apache License, Version 2.0 (the "License"); you may
 * not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */

import React from 'react';
import { render } from 'enzyme';
import { requiredProps } from '../../../test/required_props';

import {
  ALIGN_ITEMS,
  EuiPageHeader,
  EuiPageHeaderProps,
  RESPONSIVE_ORDER,
} from './page_header';

const tabs: EuiPageHeaderProps['tabs'] = [
  {
    label: 'Tab 1',
    isSelected: true,
  },
  {
    label: 'Tab 2',
  },
];

const rightSideContent: EuiPageHeaderProps['rightSideContent'] = [
  <button>Button 1</button>,
  <button>Button 2</button>,
];

describe('EuiPageHeader', () => {
  test('is rendered', () => {
    const component = render(<EuiPageHeader {...requiredProps} />);

    expect(component).toMatchSnapshot();
  });

  describe('props', () => {
    describe('pageTitle', () => {
      test('is rendered', () => {
        const component = render(<EuiPageHeader pageTitle="Page title" />);

        expect(component).toMatchSnapshot();
      });

      test('is rendered with icon', () => {
        const component = render(
          <EuiPageHeader pageTitle="Page title" iconType="logoKibana" />
        );

        expect(component).toMatchSnapshot();
      });
    });

    describe('tabs', () => {
      test('is rendered', () => {
        const component = render(<EuiPageHeader tabs={tabs} />);

        expect(component).toMatchSnapshot();
      });

      test('is rendered with tabsProps', () => {
        const component = render(
          <EuiPageHeader tabs={tabs} tabsProps={requiredProps} />
        );

        expect(component).toMatchSnapshot();
      });
    });

    describe('leftSideContent', () => {
      test('is rendered', () => {
        const component = render(
          <EuiPageHeader leftSideContent={<p>Anything</p>} />
        );

        expect(component).toMatchSnapshot();
      });
    });

    describe('description', () => {
      test('is rendered', () => {
        const component = render(<EuiPageHeader description="Description" />);

        expect(component).toMatchSnapshot();
      });
    });

    describe('rightSideContent', () => {
      test('is rendered', () => {
        const component = render(
          <EuiPageHeader rightSideContent={rightSideContent} />
        );

        expect(component).toMatchSnapshot();
      });

      test('is rendered with rightSideResponsive as true', () => {
        const component = render(
          <EuiPageHeader
            rightSideContent={rightSideContent}
            rightSideResponsive={true}
          />
        );

        expect(component).toMatchSnapshot();
      });
    });

    describe('children', () => {
      test('is rendered', () => {
        const component = render(<EuiPageHeader>Child</EuiPageHeader>);

        expect(component).toMatchSnapshot();
      });

      test('is rendered even if content props are passed', () => {
        const component = render(
          <EuiPageHeader
            pageTitle="Page title"
            tabs={tabs}
            rightSideContent={rightSideContent}>
            Child
          </EuiPageHeader>
        );

        expect(component).toMatchSnapshot();
      });
    });

    describe('alignItems', () => {
      ALIGN_ITEMS.forEach((alignment) => {
        it(`${alignment} is rendered`, () => {
          const component = render(
            <EuiPageHeader
              pageTitle="Page title"
              rightSideContent={rightSideContent}
              alignItems={alignment}
            />
          );

          expect(component).toMatchSnapshot();
        });
      });
    });

    describe('responsiveOrder', () => {
      RESPONSIVE_ORDER.forEach((order) => {
        it(`${order} is rendered`, () => {
          const component = render(
            <EuiPageHeader
              pageTitle="Page title"
              rightSideContent={rightSideContent}
              responsiveOrder={order}
            />
          );

          expect(component).toMatchSnapshot();
        });
      });
    });

    describe('responsive', () => {
      test('is rendered as false', () => {
        const component = render(<EuiPageHeader responsive={false} />);

        expect(component).toMatchSnapshot();
      });
    });

    describe('restrictWidth', () => {
      test('is rendered as true', () => {
        const component = render(<EuiPageHeader restrictWidth={true} />);

        expect(component).toMatchSnapshot();
      });
    });
  });
});
