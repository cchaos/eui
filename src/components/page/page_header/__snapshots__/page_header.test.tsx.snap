// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EuiPageHeader is rendered 1`] = `
<div
  aria-label="aria-label"
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--center testClass1 testClass2"
  data-test-subj="test subject string"
>
  Anything
</div>
`;

exports[`EuiPageHeader props alignItems bottom is rendered 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--bottom"
>
  <div
    class="euiPageHeaderContent"
  >
    <div
      class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexEnd euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
    >
      <div
        class="euiFlexItem"
      >
        <h1
          class="euiTitle euiTitle--large"
        >
          Page title
        </h1>
         
      </div>
      <div
        class="euiFlexItem euiFlexItem--flexGrowZero"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
        >
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 1
            </button>
          </div>
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 2
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeader props alignItems center is rendered 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--center"
>
  <div
    class="euiPageHeaderContent"
  >
    <div
      class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
    >
      <div
        class="euiFlexItem"
      >
        <h1
          class="euiTitle euiTitle--large"
        >
          Page title
        </h1>
         
      </div>
      <div
        class="euiFlexItem euiFlexItem--flexGrowZero"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
        >
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 1
            </button>
          </div>
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 2
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeader props alignItems top is rendered 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--top"
>
  <div
    class="euiPageHeaderContent"
  >
    <div
      class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
    >
      <div
        class="euiFlexItem"
      >
        <h1
          class="euiTitle euiTitle--large"
        >
          Page title
        </h1>
      </div>
      <div
        class="euiFlexItem euiFlexItem--flexGrowZero"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
        >
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 1
            </button>
          </div>
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 2
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeader props page content props are passed down is rendered 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--tabsAtBottom euiPageHeader--center"
>
  <div
    class="euiPageHeaderContent"
  >
    <div
      class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
    >
      <div
        class="euiFlexItem"
      >
        <h1
          class="euiTitle euiTitle--large"
        >
          <span
            class="euiPageHeaderContent__titleIcon"
            data-euiicon-type="logoKibana"
          />
          Page title
        </h1>
        <div
          class="euiSpacer euiSpacer--l"
        />
        <div
          class="euiText euiText--medium euiText--constrainedWidth"
        >
          <p>
            Description
          </p>
        </div>
      </div>
      <div
        class="euiFlexItem euiFlexItem--flexGrowZero"
      >
        <div
          class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--responsive euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
        >
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 1
            </button>
          </div>
          <div
            class="euiFlexItem euiFlexItem--flexGrowZero"
          >
            <button>
              Button 2
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="euiPageHeaderContent__bottom"
    >
      <div
        class="euiSpacer euiSpacer--l"
      />
      <p>
        Anything
      </p>
      <div
        class="euiSpacer euiSpacer--l"
      />
      <div
        aria-label="aria-label"
        class="euiTabs euiTabs--condensed testClass1 testClass2"
        data-test-subj="test subject string"
        role="tablist"
      >
        <button
          aria-selected="true"
          class="euiTab euiTab-isSelected"
          role="tab"
          type="button"
        >
          <span
            class="euiTab__content"
          >
            Tab 1
          </span>
        </button>
        <button
          aria-selected="false"
          class="euiTab"
          role="tab"
          type="button"
        >
          <span
            class="euiTab__content"
          >
            Tab 2
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeader props responsive is rendered as false 1`] = `
<div
  class="euiPageHeader euiPageHeader--center"
/>
`;

exports[`EuiPageHeader props responsive is rendered as reverse 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsiveReverse euiPageHeader--center"
/>
`;

exports[`EuiPageHeader props restrictWidth is rendered as custom 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--center euiPageHeader--restrictWidth-custom"
  style="max-width:100px"
/>
`;

exports[`EuiPageHeader props restrictWidth is rendered as true 1`] = `
<div
  class="euiPageHeader euiPageHeader--responsive euiPageHeader--center euiPageHeader--restrictWidth-default"
/>
`;

exports[`EuiPageHeaderContent is rendered 1`] = `
<div
  aria-label="aria-label"
  class="euiPageHeaderContent"
  data-test-subj="test subject string"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
  </div>
</div>
`;

exports[`EuiPageHeaderContent props alignItems bottom is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexEnd euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <h1
        class="euiTitle euiTitle--large"
      >
        Page title
      </h1>
       
    </div>
    <div
      class="euiFlexItem euiFlexItem--flexGrowZero"
    >
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 1
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 2
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props alignItems center is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <h1
        class="euiTitle euiTitle--large"
      >
        Page title
      </h1>
       
    </div>
    <div
      class="euiFlexItem euiFlexItem--flexGrowZero"
    >
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 1
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 2
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props alignItems top is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <h1
        class="euiTitle euiTitle--large"
      >
        Page title
      </h1>
    </div>
    <div
      class="euiFlexItem euiFlexItem--flexGrowZero"
    >
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 1
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 2
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props children is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
  </div>
  Child
</div>
`;

exports[`EuiPageHeaderContent props children is rendered even if content props are passed 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <h1
        class="euiTitle euiTitle--large"
      >
        Page title
      </h1>
    </div>
    <div
      class="euiFlexItem euiFlexItem--flexGrowZero"
    >
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 1
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 2
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    class="euiPageHeaderContent__bottom"
  >
    <div
      class="euiSpacer euiSpacer--l"
    />
    Child
    <div
      class="euiSpacer euiSpacer--l"
    />
    <div
      class="euiTabs euiTabs--condensed"
      role="tablist"
    >
      <button
        aria-selected="true"
        class="euiTab euiTab-isSelected"
        role="tab"
        type="button"
      >
        <span
          class="euiTab__content"
        >
          Tab 1
        </span>
      </button>
      <button
        aria-selected="false"
        class="euiTab"
        role="tab"
        type="button"
      >
        <span
          class="euiTab__content"
        >
          Tab 2
        </span>
      </button>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props description is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <div
        class="euiText euiText--medium euiText--constrainedWidth"
      >
        <p>
          Description
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props leftSideContent is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
  </div>
  <p>
    Anything
  </p>
</div>
`;

exports[`EuiPageHeaderContent props pageTitle is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <h1
        class="euiTitle euiTitle--large"
      >
        Page title
      </h1>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props pageTitle is rendered with icon 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <h1
        class="euiTitle euiTitle--large"
      >
        <span
          class="euiPageHeaderContent__titleIcon"
          data-euiicon-type="logoKibana"
        />
        Page title
      </h1>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props responsive is rendered as false 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
  </div>
</div>
`;

exports[`EuiPageHeaderContent props responsive is rendered as reverse 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
  </div>
</div>
`;

exports[`EuiPageHeaderContent props rightSideContent is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
    <div
      class="euiFlexItem euiFlexItem--flexGrowZero"
    >
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 1
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 2
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props rightSideContent is rendered with rightSideResponsive as true 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    />
    <div
      class="euiFlexItem euiFlexItem--flexGrowZero"
    >
      <div
        class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--responsive euiFlexGroup--wrap euiPageHeaderContent__rightSideContent"
      >
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 1
          </button>
        </div>
        <div
          class="euiFlexItem euiFlexItem--flexGrowZero"
        >
          <button>
            Button 2
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props tabs is rendered 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <div
        class="euiTabs euiTabs--condensed"
        role="tablist"
      >
        <button
          aria-selected="true"
          class="euiTab euiTab-isSelected"
          role="tab"
          type="button"
        >
          <span
            class="euiTab__content"
          >
            Tab 1
          </span>
        </button>
        <button
          aria-selected="false"
          class="euiTab"
          role="tab"
          type="button"
        >
          <span
            class="euiTab__content"
          >
            Tab 2
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`EuiPageHeaderContent props tabs is rendered with tabsProps 1`] = `
<div
  class="euiPageHeaderContent"
>
  <div
    class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsFlexStart euiFlexGroup--directionRow euiFlexGroup--responsive euiPageHeaderContent__top"
  >
    <div
      class="euiFlexItem"
    >
      <div
        aria-label="aria-label"
        class="euiTabs euiTabs--condensed testClass1 testClass2"
        data-test-subj="test subject string"
        role="tablist"
      >
        <button
          aria-selected="true"
          class="euiTab euiTab-isSelected"
          role="tab"
          type="button"
        >
          <span
            class="euiTab__content"
          >
            Tab 1
          </span>
        </button>
        <button
          aria-selected="false"
          class="euiTab"
          role="tab"
          type="button"
        >
          <span
            class="euiTab__content"
          >
            Tab 2
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
